### ISSUE TITLE
Actual input value in stripe form dont match what is passed to .type 

### ISSUE THREAD

**Issue by aldarund** [cypress-io/cypress#2233]
### Current behavior:

https://i.imgur.com/O5aCPFg.png

### Desired behavior:

Actual text in input matches text in cypress type command

### Steps to reproduce:

```
    cy.visit(`https://stripe.github.io/elements-examples/`)

    cy.wait(1000)

    cy.get('.StripeElement iframe').then(function($iframe) {
      const $doc = $iframe.contents()
      cy.wrap($doc.find('input[name=cardnumber]')[0]).type('4242424242424242')
    })
```

### Versions

Cypress 3.0.2
Behave same in chrome 68 and electron

**Comment by roadhousecooper** [comment]
I've had the same issue when using the .type command to input numbers. It appears that the fourth digit of the number gets moved to the last digit position every time without fail.

**Comment by Bkucera** [comment]
Hi, This should be fixed by #2016 and is pending release

**Comment by aldarund** [comment]
ye, it work fine in cypress 3.0.3
